shared-bindings/microcontroller/RunMode.c

```diff
@@ -64,6 +68,7 @@ const mcu_runmode_obj_t mcu_runmode_bootloader_obj = {
 };
 
 STATIC const mp_rom_map_elem_t mcu_runmode_locals_dict_table[] = {
+    {MP_ROM_QSTR(MP_QSTR_UF2),        MP_ROM_PTR(&mcu_runmode_uf2_obj)},
```

Comment:
Please document this. For ports where UF2 is the only bootloader we should make UF2 and BOOTLOADER equivalent.